Jobsway2goMvc/Controllers/GroupsController.cs

```diff
@@ -52,20 +296,31 @@ public IActionResult Create()
             return View();
         }
 
+        // POST: Groups/Create
+        // To protect from overposting attacks, enable the specific properties you want to bind to.
+        // For more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
         [HttpPost]
```

Comment:
In a meanwhile you can delete these types of generated comments, they are not needed.